src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -286,39 +287,6 @@ export const bankFieldsHaveInput = formData =>
     'view:bankAccount.bankName',
   ]);
 
-export const AddressViewField = ({ formData }) => {
-  const {
-    addressLine1,
-    addressLine2,
-    addressLine3,
-    city,
-    country,
-    state,
-    zipCode,
-  } = formData;
-  let zipString;
-  if (zipCode) {
-    const firstFive = zipCode.slice(0, 5);
-    const lastChunk = zipCode.length > 5 ? `-${zipCode.slice(5)}` : '';
-    zipString = `${firstFive}${lastChunk}`;
-  }
-
-  let lastLine;
-  if (country === USA) {
-    lastLine = `${city}, ${state} ${zipString}`;
-  } else {
-    lastLine = `${city}, ${country}`;
-  }
-  return (
-    <p className="blue-bar-block">
```

Comment:
The "blue-bar-block" class was not included in the global version per design decision.
- It had already been removed in the HLR design process
- Form 526 did not match this decision.